Description: Disable all optional features, reenabled in the port's Makefile.
Forwarded: not-needed
Author: Peter Pentchev <roam@FreeBSD.org>
Last-Update: 2002-02-22

--- Imakefile.orig	2014-11-11 13:01:21 UTC
+++ Imakefile
@@ -4,13 +4,13 @@
 
 
 XCOMM Remove the next line if you want genuine Xaw instead of Xaw3d
-#define XAW3D
+XCOMM define XAW3D
 
 XCOMM Remove the next line if you don't want to enable XTEST support
-#define XTEST
+XCOMM define XTEST
 
 XCOMM Remove the next line if you want to disable internationalization
-#define I18N
+XCOMM define I18N
 
 
 SRCS = xvkbd.c findwidget.c
